puppet/modules/freight/manifests/user.pp

```diff
@@ -70,6 +70,7 @@
   include apache::mod::mime
 
   web::vhost { $vhost:
+    serveraliases => [ "${vhost}-backend.theforeman.org", "${vhost}.${facts['networking']['fqdn']}" ],
```

Comment:
gotta add the right DNS entries before deploying this, or LE will freak out

Reply:
Can we make the servername `deb-backend`? I don't mind adding the `$vhost.$fqdn` as an alias.

Reply:
no, because then stagingdeb and archivedeb will also have this alias, which is not what I wanted :)

Reply:
So we can't make `$vhost` `deb-backend` instead?

My reasoning is that I'd like to use `mod_md` to get the certificate (based on the `servername`) instead of the current letsencrypt module we use, but we don't have to pull that in scope here.

Reply:
We have 3 instances of `freight::user`: `main`, `archive` and `staging` which create `deb`, `archivedeb` and `stagingdeb` vhosts.
We should switch those to produce `deb-backend`, `archivedeb-backend` and `stagingdeb-backend` at some point, yes.
But I first wanted to introduce the alias, have time to play with the CDN and *then* drop the non-`backend` vhosts.